config/application.rb

```diff
@@ -5,11 +5,9 @@
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
-require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
 require "action_view/railtie"
-require "action_cable/engine"
```

Comment:
Maybe these should be commented out? Publish had them deleted and when I did `bundle update rails` it actually did the clever thing which is it added the commented out missing bits; but I still had to think about it:

https://github.com/DFE-Digital/manage-courses-frontend/pull/636/files#diff-b1fe55db50c712fef0673345e5b9c0d9R7-R8